ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/mapper/ClaimantResponseMapperTest.java

```diff
@@ -31,82 +33,116 @@
     @Autowired
     private ClaimantResponseMapper mapper;
 
-    @Test
-    public void shouldMapClaimantAcceptanceWithCCJFormalisationFromCCD() {
-        //given
-        CCDResponseAcceptation ccdResponse = SampleData.getResponseAcceptation(CCDFormaliseOption.CCJ);
-
-        //when
-        ResponseAcceptation response = (ResponseAcceptation)mapper.from(ccdResponse).getClaimantResponse().get();
+    private Claim.ClaimBuilder claimBuilder;
 
-        //then
-        assertThat(response).isEqualTo(ccdResponse);
+    @Before
+    public void setUp() {
+        claimBuilder = Claim.builder();
```

Comment:
this can be inlined. doesn't do any specific setup.

Reply:
Setup is not doing anything except declaring a builder reference. That is not the purpose of `setup` in unit tests.
Even your less line logic doesn't seem correct.

This is old code:
```
   private Claim.ClaimBuilder claimBuilder;
  @Before
    public void setUp() {
        claimBuilder = Claim.builder();
    }
  Claim claim = claimBuilder.claimantResponse(response)
            .claimantRespondedAt(LocalDateTimeFactory.nowInLocalZone())
            .build();
```

The new would be:
```
 Claim claim =  Claim.builder().claimantResponse(response)
            .claimantRespondedAt(LocalDateTimeFactory.nowInLocalZone())
            .build();
```

Reply:
The set up is used for setting up the builder here . "Claim.builder().claimantResponse(response)" makes the line longer here .